### ISSUE TITLE
Disable stalled stream protection on EventStream operations

### ISSUE THREAD

**Issue by rcoh** [smithy-lang/smithy-rs#3382]
event stream operations should not have stalled stream protection enabled because it can result in invalid timeouts

**Comment by iampkmone** [comment]
Addressing.

**Comment by jdisanti** [comment]
I don't think we need to worry about this anymore with the new stalled stream protection implementation.

**Comment by ysaito1001** [comment]
This will be addressed by https://github.com/smithy-lang/smithy-rs/pull/4036.

**Comment by aajtodd** [comment]
Done in https://github.com/smithy-lang/smithy-rs/pull/4036
